.ci/pipeline/dr_weekly_jjb.yaml

```diff
@@ -0,0 +1,70 @@
+- job-template:
+    name: "{jjb_proj}"
+    project-type: pipeline
+    folder: pdc
+    properties:
+        - github:
+            url: "https://github.com/Mellanox/libxlio"
+        - build-discarder:
+            days-to-keep: 50
+            num-to-keep: 20
+        - inject:
+            keep-system-variables: true
+            properties-content: |
+              jjb_proj={jjb_proj}
+              WEBREPO_URL=l-pxe02_pune.nvidia.com
+    description: Do NOT edit this job through the Web GUI !
+    concurrent: true
+    sandbox: true
+    parameters:
+        - string:
+            name: "sha1"
+            default: "vNext"
+            description: "For testing PRs defaults to vNext"
+        - string:
+            name: "release_tag"
+            default: "3.40.2"
+            description: "Tag to release"
+        - string:
+            name: "release_folder"
+            default: "/auto/sw/release/sw_acceleration/xlio/dr/weekly"
+            description: "Folder to release packages into"
+        - string:
+            name: "CONF_FILE"
+            default: ".ci/pipeline/dr_weekly.yaml"
+            description: "Regex to select job config file"
+        - string:
+            name: "MAIL_TO"
+            default: "1b0f9df8.NVIDIA.onmicrosoft.com@amer.teams.ms"
```

Comment:
what is this email? for which group?

Reply:
replaced with c2a304c7.NVIDIA.onmicrosoft.com@amer.teams.ms, which is XLIO weekly builds @ PDC channel under Media DevOps group